sources/packages/api/src-sql/EducationProgramsOfferings/Clean-education-programs-offering.sql

```diff
@@ -0,0 +1,15 @@
+-- Drop lacks_study_dates from sims.education_programs_offerings. 
```

Comment:
I would prefer that the file name be more towards the `course_load` than the cleanup.